### DISCUSSION TITLE
Is there any telemetry found inside an iPhone?

### DISCUSSION THREAD

**Discussion by citycritter** [jeremyckahn/chitchatter#210] [Q&A] [answered]
I see the description mentioned anonymous features. Does this mean if I use my iPhone to livestream, the phone wouldn't log anything related to this repo? If it does, where would I locate the log history? If I wanted to make a repo similar to this, how would I label the path folders or history logs?

**Comment by jeremyckahn** [comment, marked as answer]
> Does this mean if I use my iPhone to livestream, the phone wouldn't log anything related to this repo?

That's correct. Chitchatter has no telemetry or built-in logging. The asset server (GitHub Pages), pairing servers (WebTorrent), relay server (if P2P connection can't be established), and peers you connect to will see your IP address and likely log it in one way or another, but that's true of any web app. [ISPs](https://www.top10vpn.com/guides/can-your-isp-see-your-browsing-history/) and relay servers may conceivably store the end-to-end encrypted content of your communications, but that's not something that Chitchatter or any other internet-based chat app can prevent. Assuming [WebRTC encryption](https://webrtc-security.github.io/#4.3.) does not get cracked, your communication cannot be seen by anyone other than the intended recipient.

TL;DR: As far as I know, Chitchatter is as private and anonymous as a web-based chat app can get without forgoing basic functionality.

> If I wanted to make a repo similar to this, how would I label the path folders or history logs?

I don't quite understand this question. Are you looking to fork Chitchatter and persist conversation data?

**Reply by citycritter** [reply]
Thank you for the details. I was asking because my digital information seems to be compromised and my coworkers showed heavy signs of watching and listening to me through my phone. I asked these questions to see if I could find anything that has connection history to the phone. I know they used repos to start phishing attacks to my emails (the features from certain repos look familiar to the spam attacks I've received) but I don't know what repo they may have used to spy on me..

**Reply by jeremyckahn** [reply]
Yikes, I'm sorry to hear that! Chitchatter (nor any other app) can't do much about an OS-level compromise, which it sounds like you may have. Chitchatter strives to be as transparent as possible and I can't think of anything about it that would expose you in that way.

**Reply by citycritter** [reply]
Thank you, I will keep looking!
